spec/support/helpers.rb

```diff
@@ -212,7 +212,8 @@ def gem_command(command, args = "", options = {})
         args = args.gsub(/(?=")/, "\\")
         args = %("#{args}")
       end
-      sys_exec("#{Gem.ruby} -rrubygems -S gem --backtrace #{command} #{args}")
+      gem = ENV["BUNDLE_GEM"] || "#{Gem.ruby} -rrubygems -S gem --backtrace"
```

Comment:
when won't `ruby -rrubygems -S gem` invoke ruby gems?

Reply:
`ENV["BUNDLE_GEM"]` was stored `$(srcdir)/bin/gem` from ruby source repository. In ruby core repository, It needs to works working directory without source directory.